settings/tarantool_test.go

```diff
@@ -117,7 +117,7 @@ func TestSQLDefaultEngineSetting(t *testing.T) {
 	require.Equal(t, []interface{}{[]interface{}{"sql_default_engine", "vinyl"}}, resp.Data)
 
 	// Create a space with "CREATE TABLE".
-	exec := tarantool.NewExecuteRequest("CREATE TABLE t1_vinyl(a INT PRIMARY KEY, b INT, c INT);")
+	exec := tarantool.NewExecuteRequest("CREATE TABLE T1_VINYL(a INT PRIMARY KEY, b INT, c INT);")
```

Comment:
@oleg-jukovec Are `a` and `b` left in lower case on purpose or are they overlooked?